common/build/eslint-config-fluid/src/test/mockFiles/noReleaseTagMockClass.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @public
+ */
+class MockClass {
+	/**
+	 * @internal
+	 */
+	foo(): void {}
+
+	/**
+	 * @alpha
+	 */
+	bar(): void {}
+
+	/**
+	 * @beta
+	 */
+	baz(): void {}
+
+	/**
+	 * @public
+	 */
+	bazz(): void {}
+
+	/**
+	 * Correctly implemented method with valid comment.
+	 */
+	correctValidComment(): void {}
+
+	// Correctly implemented method with a slash comment.
+	correctSlashComment(): void {}
+
+	correctNoComment(): void {}
```

Comment:
Another test case that would be worth adding would be something like:
```typescript
// @public
something(): void {}
```

To make sure we don't incorrectly trigger on slash comments with tags

Reply:
I would also recommend adding a mix of properties and methods, so we make sure we're handling both.

Reply:
<img width="284" alt="Screenshot 2023-12-13 at 13 56 15" src="https://github.com/microsoft/FluidFramework/assets/111468570/f7d2f8ec-cbe0-4e8f-acbd-750d7d1b4c0b">

is It normal for the intellisense to not recognize the `@public` tag when included inside the `//` comment?

Reply:
Yeah, it is. `@public` is a JSDoc/TSDoc concept, so intellisense parsing of normal `//` comments doesn't look for it or any other tags.